### PR TITLE
Navigate to add member page

### PR DESCRIPTION
**Pull request by ghulamhafeez**
### Implementation

* Implemented navigation to the "Add Member" page upon clicking the "Add Member" button.

### Resources

https://github.com/user-attachments/assets/0de4cc5c-0922-4fb4-a502-8c1f8ecaf9fd

### CODE REVIEW

File: src/app/members/components/MemeberHeader.tsx [hunk 1]
```diff
@@ -5,14 +5,14 @@ import { Sidebar } from "@/app/layout/components/Sidebar";
...
-import { usePathname } from "next/navigation";
+import { usePathname ,useRouter} from "next/navigation";
 import { MemberBreadCrumbs } from "./MemberBreadCrumbs";
 import { Plus } from "lucide-react";
 import { Routes } from "@/lib/routes";
 
 export const MemeberHeader = () => {
   const pathname = usePathname();
-
+  const router = useRouter();
```
Comment (Mubashar1009) [inline src/app/members/components/MemeberHeader.tsx:15]: here build is failing

File: src/app/members/components/MemeberHeader.tsx [hunk 2]
```diff
@@ -59,7 +59,7 @@ export const MemeberHeader = () => {
                 />
                 <MagnifyingGlassIcon className="absolute left-3 top-1/2 transform -translate-y-1/2 text-gray-600 w-4 h-4" />
               </div>
-              <Button data-testid="addMemberButton" className="h-7">
+              <Button data-testid="addMemberButton" className="h-7"  onClick={()=> router.push(Routes.AddMember)}>
```
Comment (Monfernape) [inline src/app/members/components/MemeberHeader.tsx:62]: How about we just wrap this up in Next.js's Link component?

File: src/app/members/components/MemeberHeader.tsx (hunk 1 above)
Comment (ghulamhafeez) [reply src/app/members/components/MemeberHeader.tsx]: Fixed

**Review by Mubashar1009 — APPROVED** [review]

File: src/app/members/components/MemeberHeader.tsx (hunk 2 above)
Comment (ghulamhafeez) [reply src/app/members/components/MemeberHeader.tsx]: Fixed

File: src/app/members/components/MemeberHeader.tsx (hunk 1 above)
Comment (ghulamhafeez) [reply src/app/members/components/MemeberHeader.tsx]: Fixed

**Pull request merged**